lib/modules/manager/helmfile/artifacts.ts

```diff
@@ -70,21 +70,23 @@ export async function updateArtifacts({
     }
 
     const cmd: string[] = [];
-    const doc = parseSingleYaml(newPackageFileContent, {
+    const docs = parseYaml(newPackageFileContent, {
       removeTemplates: true,
       customSchema: Doc,
```

Comment:
error handling should probably be filter

Reply:
?

Reply:
Right now it will throw completely if any of the docs does not fit the schema. 

see https://github.com/renovatebot/renovate/blob/main/lib/util/yaml.ts#L27